### PR TITLE
Dev agentlocation

### PR DESCRIPTION
**Pull request by HouShikon**

### CODE REVIEW

File: .gitignore
Comment (caranha) [inline .gitignore:1]: Please add the entire `.idea` folder to `.gitignore` and remove the files in this folder from the repository
(Koudou.iml, inspectionProfiles, misc.xml, etc)

Why is `exporter.py` here?

File: src/model/behavioral/simulation.py [hunk 1]
```diff
@@ -124,6 +130,17 @@ def step(self,step_length,logger):
...
 
+
+	def get_agents_location(self):
+		#record the agents location in every steps
+		agent_location = []
+		for agent in self.agents:
+			agent_location.append([agent.agent_id, agent.coordinate.get_lat_lon()])
+		# dataframe = pd.DataFrame(data=agent_location,columns=["id","coordinate"])
+		dataframe = pd.DataFrame(data=agent_location)
+		dataframe.to_csv("location.csv", mode='a', index=False, sep=",", header=False)
```
Comment (caranha) [inline src/model/behavioral/simulation.py:141]: If you generate a data frame save the location to a file every step, the simulation will be VERY slow. Also, it is not necessary to save to a DataFrame if you are going to save to a file immediately.

It is better if you cache the saved locations, and save them to a file occasionally.

I believe that the `logger` library allows you to do that. Please check that

**Review by caranha — CHANGES_REQUESTED** [review]
Do not save to a file at every agent update, that will be incredibly slow!

Did you measure the change of speed with and without this addition?

**Comment by HouShikon** [comment]
Actually I did measure the change of speed, and it turns out not a significantly change.
Probably because I used the mode='a'

**Review by caranha — APPROVED** [review]

**Pull request merged**